src/main/java/frc/team2412/robot/subsystem/ShooterSubsystem.java

```diff
@@ -195,7 +224,7 @@ public void setFlywheelVelocity(double velocity) {
      * Starts both flywheel motors
      */
     public void startFlywheel() {
-        flywheelMotor1.set(ControlMode.Velocity, FLYWHEEL_VELOCITY);
+        flywheelMotor1.set(ControlMode.Velocity, flywheelVelocityEntry.getDouble(INITIAL_FLYWHEEL_VELOCITY));
     }
```

Comment:
Should verify that `flywheelVelocityEntry` is a reasonable value before applying it

Reply:
Since we're now using constants, I don't think we need to check.